CMakeLists.txt

```diff
@@ -6,11 +6,35 @@ endif ()
 
 project (dlfcn-win32 C)
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules")
```

Comment:
Using `CMAKE_SOURCE_DIR` is not indicate because it would not work if the project is included via `add_subdirectory`, see https://github.com/dlfcn-win32/dlfcn-win32/issues/49 . I suggest to use either `CMAKE_CURRENT_SOURCE_DIR` or `PROJECT_SOURCE_DIR` .